### ISSUE TITLE
Add method to clear all invocations for a mock

### ISSUE THREAD

**Issue by maximkulkin** [kaste/mockito-python#16]
Sometimes setting up test environment is hard and might involve some invocations on a mock object. But at the end we want to check only invocations that were made after particular action without need to take into account previous invocations. It would be nice to have a method to clear all invocations once setup is done so that test can check only invocations that happened afterwards.

Example:

```python
import mockito

mock_obj = mockito.mock( ... )
obj_under_test = MyObject(mock_obj)   # this makes some invocations on mock_obj

mockito.clear(mock_obj)

obj_under_test.do_stuff()  # triggers more invocations on mock_obj that we want to test

mockito.verify(mock_obj).do_foo()
```

This feature was [merged into Java Mockito](https://github.com/mockito/mockito/pull/286).
